CONTRIBUTING.md

```diff
@@ -0,0 +1,227 @@
+
+Filing bugs
+-----------
+We use Github issues to track all bugs and feature requests; feel free to
+open an issue if you have found a bug or wish to see a feature implemented.
+
+It is recommended to check that your issue complies with the
+following rules before submitting:
+
+-  Verify that your issue is not being currently addressed by other
+   [issues](https://github.com/pydicom/pydicom/issues?q=)
+   or [pull requests](https://github.com/pydicom/pydicom/pulls?q=).
+
+-  Please ensure all code snippets and error messages are formatted in
+   appropriate code blocks.
+   See [Creating and highlighting code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks).
+
+-  Please include your operating system type and version number, as well
+   as your Python, pydicom, numpy, and scipy versions. This information
+   can be found by running the following code snippet:
+
+  ```python
+  import platform; print(platform.platform())
+  import sys; print("Python", sys.version)
+  ```
+
+-  please include a [reproducible](http://stackoverflow.com/help/mcve) code
+   snippet or link to a [gist](https://gist.github.com). If an exception is
+   raised, please provide the traceback. (use `%xmode` in ipython to use the
+   non beautified version of the trackeback)
+
+
+New contributor tips
+--------------------
+
+A great way to start contributing to pydicom is to pick an item
+from the list of [Easy issues](https://github.com/pydicom/pydicom/issues?labels=Easy)
+in the issue tracker. Resolving these issues allow you to start
+contributing to the project without much prior knowledge. Your
+assistance in this area will be greatly appreciated by the more
+experienced developers as it helps free up their time to concentrate on
+other issues.
+
+Documentation
+-------------
+
+We are glad to accept any sort of documentation: function docstrings,
+reStructuredText documents (like this one), tutorials, etc.
+reStructuredText documents live in the source code repository under the
+doc/ directory.
+
+You can edit the documentation using any text editor and then generate
+the HTML output by typing ``make html`` from the doc/ directory.
+Alternatively, ``make`` can be used to quickly generate the
+documentation without the example gallery. The resulting HTML files will
+be placed in ``_build/html/`` and are viewable in a web browser. See the
+``README`` file in the ``doc/`` directory for more information.
+
+For building the documentation, you will need
+[sphinx](http://sphinx.pocoo.org/),
+[matplotlib](http://matplotlib.org/), and
+[pillow](http://pillow.readthedocs.io/en/latest/).
+
+When you are writing documentation, it is important to keep a good
+compromise between mathematical and algorithmic details, and give
+intuition to the reader on what the algorithm does. It is best to always
+start with a small paragraph with a hand-waving explanation of what the
+method does to the data and a figure (coming from an example)
+illustrating it.
```

Comment:
I think this leans a little to heavily on algorithmic computations - in `pydicom` it is often more important to mention the relevent part of the DICOM standard.